resources/views/vendor/.gitkeep

```diff
@@ -1 +0,0 @@
-
```

Comment:
Ten folder jest na widoki, które są z bibliotek które dołączymy czy tylko mi się wydaje? To powinno zostać, albo powiniśmy to zamienić na `.gitignore` o treści:
```
!.gitignore
*
```